packages/core/content-manager/server/services/utils/__tests__/query-populate.test.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+const { getQueryPopulate } = require('../populate');
+
+const getFilterQuery = (conditions) => ({
+  filters: {
+    $or: [{
+      $and: [{
+        $or: conditions
+      }]
+    }]
+  }
+})
+
+const uid = 'model';
```

Comment:
Just a head's up, but in the future (ts improvements), we'll need to use UID format (api::string.string, admin::string, etc...) even for test data like here, otherwise APIs will complain

Reply:
will change this uid just in case!